backend/src/routes/users.js

```diff
@@ -16,4 +16,22 @@ router.post('/', (req, res) => {
     });
 });
 
+router.get('/:userId', (req, res) => {
+	User.findById(req.params.userId).then((user) => {
+		res.json(user);
+	});
+});
+
+router.put('/:userId', (req, res) => {
+	User.findByIdAndUpdate(req.params.userId, req.body, { new: true }).then((user) => {
+		res.json(user);
+	});
+});
+
+router.delete('/:userId', (req, res) => {
+	User.findByIdAndRemove(req.params.userId).then((user) => {
```

Comment:
This should eventually cascade to delete the `Preferences` whose `userID` field matches the query param `userID` that the FE passes to the URL. We can fix this once I merge the schema for it.

Reply:
So, a user needs a list of listings under their profile as well then.

Reply:
Im not sure if you mean on the FE or BE. We probably dont need the list under the user model on the BE. Currently a lister has a `listerID` which is an `ObjectID` of the user who created the listing. As such-when we need to get listings for a user, we can simply query like so:
```commandline
Listing.find({listerID: "userID"}) - userID comes from the FE
```